javascript/example/DRACOLoader.js

```diff
@@ -161,12 +174,24 @@ THREE.DRACOLoader.prototype = {
                   norAttributeData.GetValue(i + 2));
             }
         }
+
+        for (let i = 0; i < numTextureCoordinates; i += 2) {
+            // Add texture coordinates.
+            if (texCoordAttId != -1) {
```

Comment:
This line (180) should be before the for loop ( we don't want to loop through all the numTextureCoordinates when texCoordAttId is not used).